canonical/canonical.com
# WD-909 New career explorer

## Done

- Applied game redesign 

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: https://canonical-com-794.demos.haus/careers/career-explorer
- Check functionality 
- Check against design, below:
![image-20230119-172730](https://user-images.githubusercontent.com/1413534/221901698-2e8dd343-d1c5-477e-8a4c-25c238d53436.png)


## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-909
